esphome/components/improv/improv.cpp

```diff
@@ -0,0 +1,43 @@
+#include "improv.h"
+
+namespace improv {
```

Comment:
```suggestion
namespace improv {
```

What does improv stand for? improve? improvise? ...
Maybe there's a better name like `provision_improv`

Reply:
Improved Provisioning.
This folder/namespace acts as a library set of files can will be available to use for other projects too. Potentially this will become a platformio library.
I have sent you a PM